assets/clickhouse_native_json/methodology.md

```
@@ -0,0 +1,70 @@
+# ClickHouse methodology
+
+**Version: [23.3.1.2823](https://hub.docker.com/layers/clickhouse/clickhouse-server/23.3.1.2823/images/sha256-b88fd8c71b64d3158751337557ff089ff7b0d1ebf81d9c4c7aa1f0b37a31ee64?context=explore)**
+
+**File with Formatted Queries:** [Config File](/assets/clickhouse_native_json/config.yaml)
+
+## Setup
+
+### Docker   
+We benchmark in one of ClickHouse’s provided docker images `clickhouse/clickhouse-server:latest`.
+
+### Configuration  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)